models/ed/R/write.configs.ed.R

```diff
@@ -531,7 +531,9 @@ write.config.xml.ED2 <- function(settings, trait.values, defaults = settings$con
         vals <- modifyList(vals, converted.trait.values)
 
         ## Convert settings constants to ED units
-        converted.defaults <- convert.samples.ED(defaults[[pft]]$constants)
+        constants.to.convert <- defaults[[pft]]$constants
+        constants.to.convert$num <- NULL
```

Comment:
what happens if the user specifies in pecan.xml a specific ED num to use? Would it be better if we don't write the num in the webinterface?